tests/framework/installer/nfs.go

```diff
@@ -0,0 +1,202 @@
+	logger.Info("creating volume from nfs server in namespace %s", namespace)
+
+	clusterIP, err := h.GetNFSServerClusterIP(namespace)
+	if err != nil {
+		return err
+	}
+	nfsServerPV := h.installData.GetNFSServerPV(namespace, clusterIP)
+	nfsServerPVC := h.installData.GetNFSServerPVC()
+
+	logger.Info("creating nfs server pv")
+	if _, err := h.k8shelper.KubectlWithStdin(nfsServerPV, createFromStdinArgs...); err != nil {
+		return err
+	}
+
+	logger.Info("creating nfs server pvc")
+	if _, err := h.k8shelper.KubectlWithStdin(nfsServerPVC, createFromStdinArgs...); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// UninstallNFSServer uninstalls the NFS Server from the given namespace
+func (h *InstallHelper) UninstallNFSServer(systemNamespace, namespace string) {
+	logger.Infof("uninstalling nfsserver from namespace %s", namespace)
+
+	_, err := h.k8shelper.DeleteResource("pvc", "nfs-pv-claim")
+	h.checkError(err, fmt.Sprintf("cannot remove nfs pvc"))
+
+	_, err = h.k8shelper.DeleteResource("pv", "nfs-pv")
+	h.checkError(err, fmt.Sprintf("cannot remove nfs pv"))
+
+	_, err = h.k8shelper.DeleteResource("-n", namespace, "nfsservers.nfs.rook.io", namespace)
+	h.checkError(err, fmt.Sprintf("cannot remove nfsserver %s", namespace))
+
+	crdCheckerFunc := func() error {
+		_, err := h.k8shelper.RookClientset.NfsV1alpha1().NFSServers(namespace).Get(namespace, metav1.GetOptions{})
+		return err
+	}
+	err = h.waitForCustomResourceDeletion(namespace, crdCheckerFunc)
+	h.checkError(err, fmt.Sprintf("failed to wait for crd %s deletion", namespace))
+
+	_, err = h.k8shelper.DeleteResource("namespace", namespace)
+	h.checkError(err, fmt.Sprintf("cannot delete namespace %s", namespace))
+
+	logger.Infof("removing the operator from namespace %s", systemNamespace)
+	_, err = h.k8shelper.DeleteResource("crd", "nfsservers.nfs.rook.io")
+	h.checkError(err, "cannot delete CRDs")
+
+	nfsOperator := h.installData.GetNFSServerOperator(systemNamespace)
+	_, err = h.k8shelper.KubectlWithStdin(nfsOperator, deleteFromStdinArgs...)
+	h.checkError(err, "cannot uninstall rook-nfs-operator")
+
+	err = h.UninstallHostPathProvisioner()
+	h.checkError(err, "cannot uninstall hostpath provisioner")
+
+	h.k8shelper.Clientset.RbacV1beta1().ClusterRoleBindings().Delete("anon-user-access", nil)
+	logger.Infof("done removing the operator from namespace %s", systemNamespace)
+}
+
+// GatherAllNFSServerLogs gathers all NFS Server logs
+func (h *InstallHelper) GatherAllNFSServerLogs(systemNamespace, namespace, testName string) {
+	logger.Infof("Gathering all logs from NFSServer %s", namespace)
+	h.k8shelper.GetRookLogs("rook-nfs-operator", h.Env.HostType, systemNamespace, testName)
+	h.k8shelper.GetRookLogs("rook-nfs", h.Env.HostType, namespace, testName)
+}
+
+// GatherNFSServerDebuggingInfo gathers all the descriptions for pods, pvs and pvcs
+func GatherNFSServerDebuggingInfo(k8shelper *utils.K8sHelper, namespace string) {
```

Comment:
this is the exact same as the CockroachDB debugging info function, isn't it? maybe they should be consolidated into a single function?